tests/test_grid.py

```diff
@@ -482,14 +482,14 @@ def test_quadrature_grid(self):
         N = 0
         NFP = 1
         grid_quad = QuadratureGrid(L, M, N, NFP)
-        roots, weights = special.js_roots(3, 2, 2)
+        roots, weights = special.js_roots(2, 2, 2)
         quadrature_nodes = np.stack(
             [
-                np.array([roots[0]] * 5 + [roots[1]] * 5 + [roots[2]] * 5),
+                np.array([roots[0]] * 5 + [roots[1]] * 5),
```

Comment:
I guess I don't understand why roots[2] is replaced with roots[1] .

Reply:
roots[2] is the third node, which no longer exists in ``roots``

Reply:
@dpanici add a comment here or replace 5 with grid.num_theta